src/main/java/edu/harvard/iq/dataverse/ingest/IngestMessage.java

```diff
@@ -32,49 +31,21 @@
  * @author Leonid Andreev
  */
 public class IngestMessage implements Serializable {
-    public static final int INGEST_MESAGE_LEVEL_ERROR = 1; 
-    public static final int INGEST_MESAGE_LEVEL_INFO = 2;
-
     /** Creates a new instance of IngestMessage */
-    public IngestMessage()  {
-        this(INGEST_MESAGE_LEVEL_INFO);
-    }
 
-    public IngestMessage(int messageLevel)  {
-        this.messageLevel = messageLevel;
+    public IngestMessage()  {
```

Comment:
Yes. I did clean it up because it was very confusing of how the message works. I think it changed a lot over the years, but nobody cleaned it. When I was looking at it, I have lost some time to understand it. The next person should have less hard time now, since the unused code is gone.